### ISSUE TITLE
Review and revise system emails for GTDs, HCTs to support accessibility workflows

### ISSUE THREAD

**Issue by carakey** [osulp/Scholars-Archive#2829]
### Descriptive summary

In the conversations with SA partners (Office of Graduate Education, Honors College), we agreed to review notification emails that get sent to depositors during the review workflow to make sure they support changes to the process.

### Expected behavior

* The final "Deposited" notification should include, at the end of the current message, "Your document has been converted to PDF format by repository administrators. Please review the final PDF version to ensure it appears as intended. If conversion errors are noted, contact ScholarsArchive@oregonstate.edu to request changes."

**Comment by simholt** [comment]
Okay -- I wrote up text for the messages (these are just the ones that go to the user; the admin ones are fine). 

The GTD and HCT ones are in green. I don't know the formatting used in your code so I hope this makes sense. I will create a ticket to fix the other worktypes.

https://docs.google.com/document/d/1beB_3Y-oFs7G_2DPQAsctTGIUT-x-rNki37RslR7nzQ/edit?usp=sharing

**Comment by simholt** [comment]
@tu I submitted a GTD and got two messages after [Step B](https://docs.google.com/document/d/1beB_3Y-oFs7G_2DPQAsctTGIUT-x-rNki37RslR7nzQ/edit?tab=t.0) here, the approved and advanced ones:

https://ir-staging.library.oregonstate.edu/concern/graduate_thesis_or_dissertations/h415p977h

**MESSAGE ONE:**
You have a new message: ScholarsArchive@OSU Message: Deposit approved!

Your deposit: '** testing testing please ignore **' ([h415p977h](https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fir-staging.library.oregontate.edu%2Fconcern%2Fgraduate_thesis_or_dissertations%2Fh415p977h&data=05%7C02%7Csarah.imholt%40oregonstate.edu%7Cb7ed1a3e639149edcd5408de6b540347%7Cce6d05e13c5e4d6287a84c4a2713c113%7C0%7C0%7C639066205732861847%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=moQO03Ry3IA5yAg343TupVJLyXaKEcdKko7QcJENTPc%3D&reserved=0)) was approved by imholts and is now live in ScholarsArchive@OSU. Citeable URL: https://ir-staging.library.oregontate.edu/concern/graduate_thesis_or_dissertations/h415p977h If you have questions, please reply to this message or contact [ScholarsArchive@oregonstate.edu](mailto:scholarsarchive@oregonstate.edu). Thank you, ScholarsArchive@OSU Admin Oregon State University Libraries and Press

Visit [https://ir-staging.library.oregontate.edu/](https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fir-staging.library.oregontate.edu%2F&data=05%7C02%7Csarah.imholt%40oregonstate.edu%7Cb7ed1a3e639149edcd5408de6b540347%7Cce6d05e13c5e4d6287a84c4a2713c113%7C0%7C0%7C639066205732895126%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=9RbEe4zNW4yiBvkm0BtI8M8sWvxmWwOtdpMpfkhkBKI%3D&reserved=0) and go to your inbox for more info.


----
**MESSAGE TWO:**
You have a new message: ScholarsArchive@OSU Message: Deposit advanced

Your deposit: '** testing testing please ignore **' ([h415p977h](https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fir-staging.library.oregontate.edu%2Fconcern%2Fgraduate_thesis_or_dissertations%2Fh415p977h&data=05%7C02%7Csarah.imholt%40oregonstate.edu%7Cc3819e6ab7fe45cc5cb708de6b540344%7Cce6d05e13c5e4d6287a84c4a2713c113%7C0%7C0%7C639066205739661001%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=GmS0fKxmWPUfzfYWujKL95Nbved3EhWGDs1%2FSmShU9I%3D&reserved=0)) was approved by imholts. It is now live in ScholarsArchive@OSU review queue for a metadata check. You will get a message when it is live in the repository. Thank you, ScholarsArchive@OSU Admin Oregon State University Libraries and Press

Visit [https://ir-staging.library.oregontate.edu/](https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fir-staging.library.oregontate.edu%2F&data=05%7C02%7Csarah.imholt%40oregonstate.edu%7Cc3819e6ab7fe45cc5cb708de6b540344%7Cce6d05e13c5e4d6287a84c4a2713c113%7C0%7C0%7C639066205739705552%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=ZvCpUzTPvupg6EkEXgF32NdLPWICRaDLnUHDvh6YKfg%3D&reserved=0) and go to your inbox for more info.

----
I would also like the spacing to match the formatting on the document; a wall of text is hard to read.

**Comment by simholt** [comment]
This is still not working. Each of the three messages that goes to the student for GTDs and HCTs should be different. The second and third messages are back to being the same.
I think we need to shelve this for next time.

**Comment by lamtu1** [comment]
@simholt Sorry about that i thought I read it as bold the text, let me remove it quickly on that but everything good beside the bold? and the 504 error do to restart of the workflow, let me see if one it already enter the workflow, will it appear again
